core/controllers/base.py

```diff
@@ -352,7 +352,7 @@ def render_template(
             values['nav_mode'] = ''
 
         if redirect_url_on_logout is None:
-            redirect_url_on_logout = self.request.uri
+            redirect_url_on_logout = '/'
```

Comment:
I don't think this gives the correct behavior if you log out of, say, the About page? In that case the page should not change.

Reply:
Aaah, yes, then I guess I should check if URL has 'create' or not.

Reply:
Done